### ISSUE TITLE
Targeting Multiple Elements with Turbo Stream over ActionCable

### ISSUE THREAD

**Issue by matthee** [hotwired/turbo-rails#280]
[Turbo 7](https://world.hey.com/hotwired/turbo-7-0dd7a27f) introduced targeting multiple elements with `targets="#css.selectors"`. The `Turbo::Streams::TagBuilder` already supports multiple targets with `*_all` methods.

However, it is currently not possible to broadcast to multiple targets using turbo streams over ActionCable, as this method expects a `target:` parameter.

I propose the following interface:

```ruby
Turbo::StreamsChannel.broadcast_append_to(
  record_or_channel_name,
  targets: dom_target_selector,
  **rendering_options
)
```

Additionally, I think we should introduce additional `*_all` methods to `Turbo::Broadcastable` as well.

**Comment by dhh** [comment]
Sounds good. Please do take a swing.

**Comment by ccollins** [comment]
Just coming back around to this, I don't have a PR, but I was able to use this functionality browsing through the code.

```
 broadcast_action_to(
      "sessions",
      action: :replace,
      target: nil,
      targets: ".session-class-#{id}",
      partial: "users/session_status",
      locals: { user: self })
```

This did indeed broadcast to all classes, but only if I set the `target` to nil, otherwise it won't use `targets`.  I think it shouldn't be too hard to get some wrappers around these, but I just wanted to post a working follow up.

**Comment by colinambler** [comment]
@ccollins you saved me!  I had a very similar case and got through it using your post.  The crucial point, as you mention, was setting `target: nil, targets: ".#{<< your multiple-instance id>>>}"`

And corresponding in view: 
```
<%= turbo_frame_tag dom_id( xxx ) do %>
    <div class="<%= dom_id(xxx) %>">
      
    </div>
  <% end %>
```

**Comment by jaredcwhite** [comment]
@dhh Hmm, I'm not sure why this got closed? As of 1.2.0.beta2, this still feels like a bug because if you broadcast using `targets` and don't set any `target` kwarg, it fails—the only remedy is to explicitly set `target: nil`.
